server/grpc/api/runner.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"context"
+
+	"github.com/mesg-foundation/engine/config"
+	protobuf_api "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/mesg-foundation/engine/runner"
+	"github.com/mesg-foundation/engine/sdk"
+)
+
+// RunnerServer is the type to aggregate all Runner APIs.
+type RunnerServer struct {
+	sdk *sdk.SDK
+	cfg *config.Config
+}
+
+// NewRunnerServer creates a new RunnerServer.
+func NewRunnerServer(sdk *sdk.SDK, cfg *config.Config) *RunnerServer {
+	return &RunnerServer{sdk: sdk, cfg: cfg}
+}
+
+// Create creates a new runner.
+func (s *RunnerServer) Create(ctx context.Context, req *protobuf_api.CreateRunnerRequest) (*protobuf_api.CreateRunnerResponse, error) {
+	// credUsername, credPassphrase, err := GetCredentialFromContext(ctx)
```

Comment:
We can still use the `GetCredentialFromContext` here as it was merged in the context in https://github.com/mesg-foundation/engine/pull/1459